main.go

```diff
@@ -319,6 +325,19 @@ func genStaks(cCtx *cli.Context) error {
 		return err
 	}
 
+	selectedAccount := aMap[account]
+
+	var defaultRegion string
+
+	// Now check the TypeID of the selected account
+	if selectedAccount.TypeID == 1 {
+		defaultRegion = userConfig.Data.AwsDefaultCommercialRegion
+	} else if selectedAccount.TypeID == 2 {
+		defaultRegion = userConfig.Data.AwsDefaultGovcloudRegion
+	} else {
+		return fmt.Errorf("unknown genStaks account type")
```

Comment:
I don't think you need an else condition.  This will break when we support non-AWS account types.